engine/spec/components/citizens_advice_cookie_preferences/cookie_banner_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe CitizensAdviceCookiePreferences::CookieBanner, type: :component do
```

Comment:
Adding a minimal test to check that the cookie banner text changes based on locale. Once we implement the preferences page translation, we can rethink how we test the language change and whether we need feature tests in the demo app.